src/codegen.zig

```diff
@@ -517,7 +517,7 @@ pub fn generateSymbol(
                         const abi_size = math.cast(usize, typed_value.ty.abiSize(mod)) orelse
                             return error.Overflow;
                         const current_pos = code.items.len;
-                        try code.resize(current_pos + abi_size);
+                        try code.writer().writeByteNTimes(0, abi_size);
```

Comment:
```suggestion
                        try code.appendNTimes(0, abi_size);
```

Compare the implementations:
* [ArrayList appendNTimes](https://ziglang.org/documentation/master/std/#std.array_list.ArrayListAligned.appendNTimes)
* [Writer appendNTimes](https://ziglang.org/documentation/master/std/#std.io.Writer.writeByteNTimes)

Reply:
TIL we had `appendNTimes` and `appendNTimesAssumeCapacity`, sigh...

Reply:
Oh lol, it was used in the snipped above the one I changed.

Reply:
yeah in general, it's better to operate directly on an ArrayList if you can; using writer() is mainly intended for adapting for a Writer-based API.